app/controllers/application_controller.rb

```diff
@@ -1549,25 +1666,33 @@ def extra_gc
 
   def log_memory_usage
     sd = sc = pd = pc = 0
+
     File.new("/proc/#{$PROCESS_ID}/smaps").each_line do |line|
-      if line.match(/\d+/)
-        val = $&.to_i
-        line.match(/^Shared_Dirty/) ? (sd += val) :
-        line.match(/^Shared_Clean/) ? (sc += val) :
-        line.match(/^Private_Dirty/) ? (pd += val) :
-        line.match(/^Private_Clean/) ? (pc += val) : 1
+      next unless line =~ /\d+/
+      val = $&.to_i
+      case line.match(/^[\w]+/)[0]
+      when "Shared_Dirty"  then sd += val
+      when "Shared_Clean"  then sc += val
+      when "Private_Dirty" then pd += val
+      when "Private_Clean" then pc += val
+      else 1
```

Comment:
I'm pretty sure it does nothing.  As mentioned elsewhere, I think this would be better as a dictionary where the keys are "Shared_DIrty" etc., and the values are the counts rather than separate local variables.  It could even be a little counter object.  I would also get rid of $& and use a match object.  I'm surprised Rubocop doesn't complain about that.